main.go

```diff
@@ -56,17 +80,24 @@ func (h spaHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
+	tracer, closer := initJaeger("manager")
+	defer closer.Close()
 	router := mux.NewRouter()
 
 	router.HandleFunc("/api/health", func(w http.ResponseWriter, r *http.Request) {
+
+		span := tracer.StartSpan("health")
+		defer span.Finish()
 		// an example API handler
 		json.NewEncoder(w).Encode(map[string]bool{"ok": true})
 	})
 	var conferenceService conferenceService
+	conferenceService.tracer = tracer
+
 	server := otohttp.NewServer()
-	RegisterConferenceService(server, conferenceService)
-	router.Handle("/oto/", server)
-	spa := spaHandler{staticPath: "build", indexPath: "index.html"}
+	RegisterConferenceService(tracer, server, conferenceService)
```

Comment:
The generate handler has only 2 params right now.
Are you sure this will work now?

Reply:
shoot, no it won't